decidim-proposals/spec/features/proposals_spec.rb

```diff
@@ -730,6 +730,22 @@
           expect(page).to have_selector("#proposals .card-grid .column:last-child", text: older_proposal.title)
         end
       end
+
+      context "randomly" do
+        let!(:lucky_proposal) { create(:proposal, feature: feature) }
+        let!(:unlucky_proposal) { create(:proposal, feature: feature) }
```

Comment:
This is previous code I just moved around. But you're right that these names no longer make sense. I used them previously when I was forcing an order, now it does not make sense in the current specs. I'll fix it.